src/current/_includes/releases/v23.2/v23.2.13.md

```diff
@@ -0,0 +1,88 @@
+      - `changefeed.buffer_entries_mem.*`
+      - `changefeed.buffer_pushback_nanos.*`
+
+      [#131417][#131417]
+- Timers have been added to key parts of the [changefeed]({% link v23.2/change-data-capture-overview.md %}) pipeline to help debug problems with feeds. [Metrics]({% link v23.2/metrics.md %}) `changefeed.stage.{stage}.latency` now emit latency histograms for each stage. The changefeed `scope` label allows debugging of a specific changefeed. [#131428][#131428]
+
+<h3 id="v23-2-13-operational-changes">Operational changes</h3>
+
+- The new [metric]({% link v23.2/metrics.md %}) `ranges.decommissioning` shows the number of ranges with a replica on a [decommissioning node]({% link v23.2/node-shutdown.md %}). [#130251][#130251]
+- The following new [metrics]({% link v23.2/metrics.md %}) show the number of RPC TCP connections established to remote nodes:
+      - `rpc.connection.connected`: the number of gRPC TCP level connections established to remote nodes.
+      - `rpc.client.bytes.egress`: the number of TCP bytes sent over gRPC on connections initiated by the cluster.
+      - `rpc.client.bytes.ingress`: the number of TCP bytes received over gRPC on connections initiated by the cluster.
+
+      [#130521][#130521]
+- Added a new configuration parameter, `server.cidr_mapping_url`, which maps IPv4 CIDR blocks to arbitrary tag names. [#130528][#130528]
+- The [metrics]({% link v23.2/metrics.md %}) `sql.bytesin` and `sql.bytesout` are now aggregate metrics if child metrics are enabled. [#130528][#130528]
+- The following new [metrics]({% link v23.2/metrics.md %}) track the number of bytes sent by an individual [changefeed]({% link v23.2/change-data-capture-overview.md %}) to each sink:
+    - `changefeed.network.bytes_in`
+    - `changefeed.network.bytes_out`
+
+    [#130664][#130664]
+- You can now set the log format for the `STDERR` changefeed sink using the `format` field in the `stderr` sink section of the [logging]({% link v23.2/logging.md %}) configuration. [#131533][#131533]
+
+<h3 id="v23-2-13-db-console-changes">DB Console changes</h3>
+
+- The [DB Console]({% link v23.2/ui-overview.md %}) now shows an alert message if the cluster's [enterprise license]({% link v23.2/licensing-faqs.md %}) is expired or will expire in fewer than 15 days, or if the cluster has no enterprise license set. [#130425][#130425]
+
+<h3 id="v23-2-13-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where the command `SHOW CLUSTER SETTING FOR VIRTUAL CLUSTER` would erroneously return `NULL` for some settings. [#128782][#128782]
+- Fixed a bug where a node could fail to start with the error `could not insert session ...: unexpected value` if an ambiguous result error occurred while inserting data into the `sqlliveness` table. [#130343][#130343]
+- Fixed a bug that could prevent [upgrade finalization]({% link v23.2/upgrade-cockroach-version.md %}) due to the upgrade pre-condition for repairing descriptor corruption. [#130519][#130519]
```

Comment:
Not going to get picky about aligning all of these with my PR, unless it's a very helpful change. And soon enough, we won't need to worry about such divergence across backports.